components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Storage/PublishedNodesJobConverterTests.cs

```diff
@@ -726,7 +726,7 @@ private class StandaloneIdentity : IIdentity {
         ""OpcNodes"": [
             {
                 ""Id"": ""i=2258"",
-                ""HeartbeatIntervalTimespan"": ""00:00:01.500""
+                ""HeartbeatInterval"": 1500
```

Comment:
>HeartbeatInterval [](http://example.com/codeflow?start=18&length=17)

let's keep the timespan test here. The test with interval is seconds is above

Reply:
Reverted changes to ` OpcNodeModel.HeartbeatInterval`.